src/frontends/tensorflow_common/src/utils.cpp

```diff
@@ -6,65 +6,67 @@
 
 #include <limits>
 
+#include "common_op_table.hpp"
 #include "openvino/opsets/opset10.hpp"
 #include "openvino/opsets/opset8.hpp"
-#include "openvino_conversions.hpp"
 
 using namespace ov;
 using namespace ov::op;
 using namespace ov::opset10;
-using namespace ov::opset8;
```

Comment:
By default, it uses opset10. For usage of opset8, we need explicitly use `opset8`. If I leave both `using namespace`, there is can be confusion about what is default.
In this PR I just moved all util routines into one place without changing opset that can be risky.

Reply:
Looks like you didn't get my point of alignment. I'm ok if you want to introduce some "default opset version" (but it looks awful from my point of view). If you expect to use "default opset" - why do you use explicit notation "opset10::Squeeze" in other place? Use then just Squeeze. Where is "wrong" place?

If it has more complicated logic in naming - better to reduce complexity and use common approach with [ov::]opsetN::Operation to be clear each time.

Reply:
I agree with you. I would rather re-do this piece and to move the whole code to opset10. Now there is some mess there:)
Will do it in the separate PR because this PR is dedicated to avoid `Squeeze` without axis and combining `openvino_conversion` and `util`

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/16656